ports/shrubnaut/Shrubnaut.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/shrubnaut"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$GAMEDIR/utils/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then 
+  source "${controlfolder}/libgl_${CFW_NAME}.txt"
+else
+  source "${controlfolder}/libgl_default.txt"
+fi
+
+# Patch game
+cd "$GAMEDIR"
+# If "gamedata/data.win" exists and its MD5 checksum matches the specified value, apply the xdelta3 patch
+if [ -f "./gamedata/data.win" ]; then
```

Comment:
Will this patch the game.win, game.unx files like below? What happens if the md5sum doesn't match? The script just continues. 🤔

Reply:
the correct source will only be the data.win file and this would happen on first start up... hence after patching then it will be converted to an updated data.win before being renamed to game.droid and no further patches upon subsequent running of the bash script